bigtop-packages/src/common/oozie/patch4-commons-lang3.diff

```diff
@@ -0,0 +1,501 @@
+diff --git a/core/src/main/java/org/apache/oozie/CoordinatorEngine.java b/core/src/main/java/org/apache/oozie/CoordinatorEngine.java
+index cec634790..c5b1f091e 100644
+--- a/core/src/main/java/org/apache/oozie/CoordinatorEngine.java
++++ b/core/src/main/java/org/apache/oozie/CoordinatorEngine.java
+@@ -35,7 +35,7 @@ import java.util.Map;
+ import java.util.Set;
+ import java.util.StringTokenizer;
+ 
+-import org.apache.commons.lang.StringUtils;
++import org.apache.commons.lang3.StringUtils;
```

Comment:
This was addressed by [OOZIE-3446](https://issues.apache.org/jira/browse/OOZIE-3446). Bumping Oozie to 5.2.0 could be an alternative fix.